.gitignore

```diff
@@ -125,4 +125,4 @@ Carthage
 *.log
 
 # docfx
-csharp/_site
\ No newline at end of file
+csharp/doc/_site
```

Comment:
I think we should move this to `csharp/.gitignore`

Reply:
yes fixed